src/platform_impl/linux/mod.rs

```diff
@@ -122,9 +122,9 @@ pub(crate) static X11_BACKEND: Lazy<Mutex<Result<Arc<XConnection>, XNotSupported
     Lazy::new(|| Mutex::new(XConnection::new(Some(x_error_callback)).map(Arc::new)));
 
 #[derive(Debug, Clone)]
-pub enum OsError {
+pub(crate) enum OsError {
```

Comment:
I like using `pub(crate)` because it lets me statically make sure I'm not exporting anything publicly that I shouldn't be exporting (I don't want to make `x11rb` a public dependency if I can help it).

Reply:
Well, the module is private and there's no desire to over make it public, though, there's no internal policy wrt such changes.